Frontend/ui-customer/src/Flow.purs

```diff
@@ -284,18 +284,33 @@ toggleSplash =
       liftFlowBT $ terminateUI $ Just "SplashScreen"
 
 currentFlowStatus :: FlowBT String Unit
-currentFlowStatus = do  
+currentFlowStatus = do
+  logField_ <- lift $ lift $ getLogFields
   liftFlowBT $ markPerformance "CURRENT_FLOW_STATUS"
   void $ lift $ lift $ toggleLoader false
   liftFlowBT $ markPerformance "VERIFY_PROFILE_CALL_API"
   verifyProfile "LazyCheck"
   liftFlowBT $ markPerformance "FLOW_STATUS_CALL_API"
   flowStatus <- Remote.flowStatusBT "LazyCheck"
   liftFlowBT $ markPerformance "RIDE_LIST_CALL_API"
+  let appName = case getAppName of
```

Comment:
@ashkritidewan Instead of case match fetch app name from window

Reply:
The app name is being fetched from window, the case match is for the naming convention of the event